src/skins/vector/skindex.js

```diff
@@ -23,13 +23,15 @@ limitations under the License.
 
 var skin = {};
 
+// TODO: Fix this so matrix-react-sdk stuff is in react SDK.
+skin['avatars.RoomAvatar'] = require('matrix-react-sdk/lib/components/views/avatars/RoomAvatar');
+skin['avatars.MemberAvatar'] = require('matrix-react-sdk/lib/components/views/avatars/MemberAvatar');
+
```

Comment:
OK, this will be clobbered when this file is regenerated but I'm about to go & fix this generation so let's go with this for now.